packages/frontend/src/pages/Signup.tsx

```diff
@@ -62,6 +115,15 @@ const Signup: FC = () => {
 
     const allowPasswordAuthentication =
         !health.data?.auth.disablePasswordAuthentication;
+    const isLinkExpired = inviteLinkQuery?.data?.expiresAt
```

Comment:
If the link expired the endpoint response is a 404 error with name: 'NotExistsError' and message 'Invite link expired', and if we can't find the invite the message 'Invite link not found'. 

We could change the error to 406 with name: 'ExpiredError' and you can adjust the FE logic to handle the error correctly. ( I can push a commit with the BE changes to this branch )

Also, related to this, if the token isn't valid we show the wrong ui.

Eg: I made up a invite code
![Screenshot 2022-06-14 at 16 15 21](https://user-images.githubusercontent.com/9117144/173613876-1bb3e440-cbe9-4aaf-8df3-a1fb3689254b.png)